build_all/base-libraries/AzureIoTUtility/src/samd/sample_init.cpp

```diff
@@ -9,7 +9,16 @@
 #include <time.h>
 #include <sys/time.h>
 #include <SPI.h>
+
+#if WIO_TERMINAL
+#include <WiFi.h>
+#include "WiFiClientSecure.h"
+static WiFiClientSecure sslClient; // for Wio Terminal variant of SAMD
+#else
```

Comment:
The Wio Terminal WiFi header file is WiFi.h, and the SSL client comes from WiFiClientSecure.h.